### DISCUSSION TITLE
Is it possible to compact Metrics older than x days further in backend stroage?

### DISCUSSION THREAD

**Discussion by MarcelGlaub** [grafana/mimir#7304] [Help and support]
In my setup remotely written Prometheus metrics to mimir, metrics are compacted and stored in MinIO as long term storage. Is it possible that mimir further compresses metrics older than 90 days in MinIO e.g. from cpu usage per second to cpu usage per minute?

**Comment by pstibrany** [comment]
Hello. What you're describing is downsampling, and Mimir doesn't have such feature.

There are some discussions about it in https://github.com/grafana/mimir/discussions/1834, but no plans at this point. If you want to continue discussion, please use that linked issue.
